### ISSUE TITLE
doNotWaitForEmptyEventLoop `onError` doesn't pass a handler.error

### ISSUE THREAD

**Issue by sviatoslav-lebediev** [middyjs/middy#318]
doNotWaitForEmptyEventLoop middleware doesn't pass a handler.error as a param. 
Here is a current code 
```
  const disableEmptyEventLoopWait = (handler, next) => {
    handler.context.callbackWaitsForEmptyEventLoop = false
    next()
  }
```

So when I use `.use(doNotWaitForEmptyEventLoop({ runOnError: true }))` my functions always finish successfully

**Comment by KnupMan** [comment]
The same to me

**Comment by KnupMan** [comment]
@vladgolubev I would really appreciate if you can find a time to fix that!

**Comment by vladgolubev** [comment]
Hey @sviatoslav-lebediev @KnupMan 

This has been fixed in version `1.0.0-alpha.39`
